src/tribler/core/components/metadata_store/restapi/search_endpoint.py

```diff
@@ -1,26 +1,29 @@
-from aiohttp import web
+from collections import defaultdict
+from typing import Dict, List
 
+from aiohttp import web
 from aiohttp_apispec import docs, querystring_schema
-
 from ipv8.REST.schema import schema
-
 from marshmallow.fields import Integer, String
-
 from pony.orm import db_session
 
+from tribler.core.components.metadata_store.db.serialization import SNIPPET
 from tribler.core.components.metadata_store.db.store import MetadataStore
 from tribler.core.components.metadata_store.restapi.metadata_endpoint import MetadataEndpointBase
 from tribler.core.components.metadata_store.restapi.metadata_schema import MetadataParameters, MetadataSchema
 from tribler.core.components.restapi.rest.rest_endpoint import HTTP_BAD_REQUEST, RESTResponse
+from tribler.core.components.knowledge.db.knowledge_db import ResourceType
 from tribler.core.utilities.utilities import froze_it
 
+SNIPPETS_TO_SHOW = 10          # The number of snippets we return from the search results
```

Comment:
I suggest to lower this number for the first iteration, for example, to 3.

Reply:
Done